fdbrpc/LoadPlugin.h

```diff
@@ -20,13 +20,14 @@
 
 #pragma once
 
+// Will need to be defined at link time
+extern "C" void *get_plugin(const char *plugin_type_name_and_version);
+
 template <class T>
 Reference<T> loadPlugin( std::string const& plugin_name ) {
-	void* plugin = loadLibrary( plugin_name.c_str() );
-	void *(*get_plugin)(const char*) = (void*(*)(const char*))loadFunction( plugin, "get_plugin" );
-
-	if ( get_plugin )
-		return Reference<T>( (T*)get_plugin( T::get_plugin_type_name_and_version() ) );
-	else
-		return Reference<T>( NULL );
+#ifdef __TLS_DISABLED__
+	return Reference<T>( NULL );
```

Comment:
and check plugin_name == fdb-libressl-plugin, or something, and then call get_tls_plugin if it matches and get_plugin if it doesn't.

We should either leave the plugin interface as a generic working plugin interface, and make it special case out the TLS plugin that's now built-in, or just remove the generic plugin interface entirely.  But I'm kind of against just crippling it whenever we aren't using the TLS code.